site/.storybook/preview.jsx

```diff
@@ -89,3 +72,25 @@ export const parameters = {
     },
   },
 };
+
+function withQuery(Story, { parameters }) {
+  const queryClient = new QueryClient({
+    defaultOptions: {
+      queries: {
+        staleTime: Infinity,
+      },
+    },
+  });
+
+  if (parameters.queries) {
+    parameters.queries.forEach((query) => {
+      queryClient.setQueryData(query.key, query.data);
```

Comment:
what are we doing here?

Reply:
We are setting the default values, passed via story parameters, for the react-query queries so the stories don't need to fetch the data. It is used to make the story independent of a BE or API.